detectors/EICDetector/Fun4All_G4_EICDetector.C

```diff
@@ -100,12 +101,12 @@ int Fun4All_G4_EICDetector(
   // And/Or read generated particles from file
 
   // eic-smear output
-  //  Input::READEIC = true;
+  Input::READEIC = true;
   INPUTREADEIC::filename = inputFile;
 
   // HepMC2 files
   //  Input::HEPMC = true;
-  Input::VERBOSITY = 0;
+  Input::VERBOSITY = 10;
```

Comment:
Thanks for the update! Could you step back the commit that changes the default behavior of the macro: `const string &inputFile`, `Input::READEIC`, `Input::SIMPLE`, `Input::VERBOSITY`, etc. This help preserve behavior of users and avoids merge conflicts.